src/main/java/frc/util/control/PIDConfig.java

```diff
@@ -43,37 +64,57 @@ public PIDConfig(double p, double i, double d, double f) {
       kIPub = pidTable.getDoubleTopic("kI").publish();
       kDPub = pidTable.getDoubleTopic("kD").publish();
       kFPub = pidTable.getDoubleTopic("kF").publish();
-      
+      kGPub = pidTable.getDoubleTopic("kG").publish();
+      kSPub = pidTable.getDoubleTopic("kS").publish();
+      kVPub = pidTable.getDoubleTopic("kV").publish();
+      kAPub = pidTable.getDoubleTopic("kA").publish();
```

Comment:
I think we can leave out the pubsub stuff for feedforward gains... at least for now. These shouldn't be tuned per se but instead calculated by sysid based on data we collect.